lib/LaTeXML/Package/braket.sty.ltxml

```diff
@@ -75,53 +75,53 @@ DefMath('\lx@Braket@{}', '\left\langle#1\right\rangle',
 # seperate macros since single vs double bars could have different meanings?
 #(but we assume the same for now)
 DefMath('\lx@braket@V{}{}', '\langle#1\,|\,#2\rangle',
-  meaning => 'inner-product',
-  alias   => '\braket');
+  meaning   => 'inner-product',
+  reversion => '\braket{#1|#2}');
 DefMath('\lx@braket@D{}{}', '\langle#1\,\|\,#2\rangle',
-  meaning => 'inner-product',
-  alias   => '\braket');
+  meaning   => 'inner-product',
+  reversion => '\braket{#1|#2}');
```

Comment:
The "D" (double) versions should revert to `\|`, rather than simply `|` (for the "V" versions).
Same comment applies below.

Reply:
Thank you for catching that. It should be enacted now.

I also rebased to the latest master, for a simpler merge.